{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 0, "total_error": 4}, "tests": {"0": {"status": "ERROR", "message": "launch_browser = <function launch_browser.<locals>.launch at 0x108c95ca0>\n\n    @pytest.fixture(scope=\"session\")\n    def browser(launch_browser: Callable[[], Browser]) -> Generator[Browser, None, None]:\n>       browser = launch_browser()\n\n/usr/local/lib/python3.9/site-packages/pytest_playwright/pytest_playwright.py:183: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.9/site-packages/pytest_playwright/pytest_playwright.py:175: in launch\n    browser = browser_type.launch(**launch_options)\n/usr/local/lib/python3.9/site-packages/playwright/sync_api/_generated.py:11345: in launch\n    self._sync(\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_sync_base.py:111: in _sync\n    return task.result()\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_browser_type.py:88: in launch\n    return from_channel(await self._channel.send(\"launch\", params))\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_connection.py:39: in send\n    return await self.inner_send(method, params, False)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Channel object at 0x108c62850>, method = 'launch', params = {}, return_as_dict = False\n\n    async def inner_send(\n        self, method: str, params: Optional[Dict], return_as_dict: bool\n    ) -> Any:\n        if params is None:\n            params = {}\n        callback = self._connection._send_message_to_server(self._guid, method, params)\n        if self._connection._error:\n            error = self._connection._error\n            self._connection._error = None\n            raise error\n        done, _ = await asyncio.wait(\n            {\n                self._connection._transport.on_error_future,\n                callback.future,\n            },\n            return_when=asyncio.FIRST_COMPLETED,\n        )\n        if not callback.future.done():\n            callback.future.cancel()\n>       result = next(iter(done)).result()\nE       playwright._impl._api_types.Error: Executable doesn't exist at /Users/openmindschooling/Library/Caches/ms-playwright/chromium-939194/chrome-mac/Chromium.app/Contents/MacOS/Chromium\nE       \u2554\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2557\nE       \u2551 Looks like Playwright Test or Playwright was just installed or updated. \u2551\nE       \u2551 Please run the following command to download new browsers:              \u2551\nE       \u2551                                                                         \u2551\nE       \u2551     playwright install                                                  \u2551\nE       \u2551                                                                         \u2551\nE       \u2551 <3 Playwright Team                                                      \u2551\nE       \u255a\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u255d\n\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_connection.py:63: Error\n", "test_name": "test_loginpage_title[chromium]", "rerun": "0"}, "1": {"status": "ERROR", "message": "launch_browser = <function launch_browser.<locals>.launch at 0x108c95ca0>\n\n    @pytest.fixture(scope=\"session\")\n    def browser(launch_browser: Callable[[], Browser]) -> Generator[Browser, None, None]:\n>       browser = launch_browser()\n\n/usr/local/lib/python3.9/site-packages/pytest_playwright/pytest_playwright.py:183: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.9/site-packages/pytest_playwright/pytest_playwright.py:175: in launch\n    browser = browser_type.launch(**launch_options)\n/usr/local/lib/python3.9/site-packages/playwright/sync_api/_generated.py:11345: in launch\n    self._sync(\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_sync_base.py:111: in _sync\n    return task.result()\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_browser_type.py:88: in launch\n    return from_channel(await self._channel.send(\"launch\", params))\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_connection.py:39: in send\n    return await self.inner_send(method, params, False)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Channel object at 0x108c62850>, method = 'launch', params = {}, return_as_dict = False\n\n    async def inner_send(\n        self, method: str, params: Optional[Dict], return_as_dict: bool\n    ) -> Any:\n        if params is None:\n            params = {}\n        callback = self._connection._send_message_to_server(self._guid, method, params)\n        if self._connection._error:\n            error = self._connection._error\n            self._connection._error = None\n            raise error\n        done, _ = await asyncio.wait(\n            {\n                self._connection._transport.on_error_future,\n                callback.future,\n            },\n            return_when=asyncio.FIRST_COMPLETED,\n        )\n        if not callback.future.done():\n            callback.future.cancel()\n>       result = next(iter(done)).result()\nE       playwright._impl._api_types.Error: Executable doesn't exist at /Users/openmindschooling/Library/Caches/ms-playwright/chromium-939194/chrome-mac/Chromium.app/Contents/MacOS/Chromium\nE       \u2554\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2557\nE       \u2551 Looks like Playwright Test or Playwright was just installed or updated. \u2551\nE       \u2551 Please run the following command to download new browsers:              \u2551\nE       \u2551                                                                         \u2551\nE       \u2551     playwright install                                                  \u2551\nE       \u2551                                                                         \u2551\nE       \u2551 <3 Playwright Team                                                      \u2551\nE       \u255a\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u255d\n\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_connection.py:63: Error\n", "test_name": "test_signup_link_visible[chromium]", "rerun": "0"}, "2": {"status": "ERROR", "message": "launch_browser = <function launch_browser.<locals>.launch at 0x108c95ca0>\n\n    @pytest.fixture(scope=\"session\")\n    def browser(launch_browser: Callable[[], Browser]) -> Generator[Browser, None, None]:\n>       browser = launch_browser()\n\n/usr/local/lib/python3.9/site-packages/pytest_playwright/pytest_playwright.py:183: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.9/site-packages/pytest_playwright/pytest_playwright.py:175: in launch\n    browser = browser_type.launch(**launch_options)\n/usr/local/lib/python3.9/site-packages/playwright/sync_api/_generated.py:11345: in launch\n    self._sync(\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_sync_base.py:111: in _sync\n    return task.result()\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_browser_type.py:88: in launch\n    return from_channel(await self._channel.send(\"launch\", params))\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_connection.py:39: in send\n    return await self.inner_send(method, params, False)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Channel object at 0x108c62850>, method = 'launch', params = {}, return_as_dict = False\n\n    async def inner_send(\n        self, method: str, params: Optional[Dict], return_as_dict: bool\n    ) -> Any:\n        if params is None:\n            params = {}\n        callback = self._connection._send_message_to_server(self._guid, method, params)\n        if self._connection._error:\n            error = self._connection._error\n            self._connection._error = None\n            raise error\n        done, _ = await asyncio.wait(\n            {\n                self._connection._transport.on_error_future,\n                callback.future,\n            },\n            return_when=asyncio.FIRST_COMPLETED,\n        )\n        if not callback.future.done():\n            callback.future.cancel()\n>       result = next(iter(done)).result()\nE       playwright._impl._api_types.Error: Executable doesn't exist at /Users/openmindschooling/Library/Caches/ms-playwright/chromium-939194/chrome-mac/Chromium.app/Contents/MacOS/Chromium\nE       \u2554\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2557\nE       \u2551 Looks like Playwright Test or Playwright was just installed or updated. \u2551\nE       \u2551 Please run the following command to download new browsers:              \u2551\nE       \u2551                                                                         \u2551\nE       \u2551     playwright install                                                  \u2551\nE       \u2551                                                                         \u2551\nE       \u2551 <3 Playwright Team                                                      \u2551\nE       \u255a\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u255d\n\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_connection.py:63: Error\n", "test_name": "test_login[chromium]", "rerun": "0"}, "3": {"status": "ERROR", "message": "launch_browser = <function launch_browser.<locals>.launch at 0x108c95ca0>\n\n    @pytest.fixture(scope=\"session\")\n    def browser(launch_browser: Callable[[], Browser]) -> Generator[Browser, None, None]:\n>       browser = launch_browser()\n\n/usr/local/lib/python3.9/site-packages/pytest_playwright/pytest_playwright.py:183: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.9/site-packages/pytest_playwright/pytest_playwright.py:175: in launch\n    browser = browser_type.launch(**launch_options)\n/usr/local/lib/python3.9/site-packages/playwright/sync_api/_generated.py:11345: in launch\n    self._sync(\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_sync_base.py:111: in _sync\n    return task.result()\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_browser_type.py:88: in launch\n    return from_channel(await self._channel.send(\"launch\", params))\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_connection.py:39: in send\n    return await self.inner_send(method, params, False)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Channel object at 0x108c62850>, method = 'launch', params = {}, return_as_dict = False\n\n    async def inner_send(\n        self, method: str, params: Optional[Dict], return_as_dict: bool\n    ) -> Any:\n        if params is None:\n            params = {}\n        callback = self._connection._send_message_to_server(self._guid, method, params)\n        if self._connection._error:\n            error = self._connection._error\n            self._connection._error = None\n            raise error\n        done, _ = await asyncio.wait(\n            {\n                self._connection._transport.on_error_future,\n                callback.future,\n            },\n            return_when=asyncio.FIRST_COMPLETED,\n        )\n        if not callback.future.done():\n            callback.future.cancel()\n>       result = next(iter(done)).result()\nE       playwright._impl._api_types.Error: Executable doesn't exist at /Users/openmindschooling/Library/Caches/ms-playwright/chromium-939194/chrome-mac/Chromium.app/Contents/MacOS/Chromium\nE       \u2554\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2557\nE       \u2551 Looks like Playwright Test or Playwright was just installed or updated. \u2551\nE       \u2551 Please run the following command to download new browsers:              \u2551\nE       \u2551                                                                         \u2551\nE       \u2551     playwright install                                                  \u2551\nE       \u2551                                                                         \u2551\nE       \u2551 <3 Playwright Team                                                      \u2551\nE       \u255a\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u2550\u255d\n\n/usr/local/lib/python3.9/site-packages/playwright/_impl/_connection.py:63: Error\n", "test_name": "test_check_login_succesful[chromium]", "rerun": "0"}}, "suite_name": "test_login.py"}}}, "date": "January 03, 2022", "start_time": 1641241444.101774, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "0", "skip": "0", "error": "4", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "4"}